README.md

```diff
@@ -1 +1,69 @@
-# oop-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="https://github.com/microverseinc/readme-template/raw/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 OOP School Library <a name="about-project"></a>
+
+> This app records all books that are in the library and their borrowers. It allows the user to add new students or teachers, add new books, save records of who borrowed a given book and when.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- [Ruby](https://www.ruby-lang.org/en/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- [Install Ruby](https://www.ruby-lang.org/en/documentation/installation/)
+
```

Comment:
- [OPTIONAL] It will be nice to write out explicit steps on how to clone and run your project locally and this will be of help to non technical users and some of them are potential recruiters.